### PR TITLE
Install updated weave CNI plugin for our vagrant environment

### PR DESCRIPTION
**Pull request by cynepco3hahue**
Signed-off-by: Lukianov Artyom <alukiano@redhat.com>

### CODE REVIEW

File: cluster/vagrant/setup_kubernetes_master.sh [hunk 1]
```diff
@@ -44,8 +44,9 @@ done
 
 set -e
 
-if [ "$NETWORK_PROVIDER" == "weave" ]; then 
-  kubectl apply -f https://github.com/weaveworks/weave/releases/download/v1.9.4/weave-daemonset-k8s-1.6.yaml
+if [ "$NETWORK_PROVIDER" == "weave" ]; then
+  kubever=$(kubectl version | base64 | tr -d '\n')
+  kubectl apply -f "https://cloud.weave.works/k8s/net?k8s-version=$kubever"
```
Comment (rmohr) [inline cluster/vagrant/setup_kubernetes_master.sh]: That is nice!

Comment (cynepco3hahue) [reply cluster/vagrant/setup_kubernetes_master.sh]: @rmohr From here https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/#pod-network :smile:

**Comment by rmohr** [comment]
ok to test

**Comment by cynepco3hahue** [comment]
@rmohr Can we merge it?

**Pull request merged**